Wrappers/Python/ccpi/astra/operators/AstraProjector3DSimple.py

```diff
@@ -65,5 +64,5 @@ def range_geometry(self):
     
     def norm(self):
         x0 = self.volume_geometry.allocate('random')
-        self.s1, sall, svec = PowerMethodNonsquare(self, 50, x0)
+        self.s1, sall, svec = LinearOperator.PowerMethodNonsquare(self, 50, x0)
```

Comment:
Could `self` be used instead, if so, I think that would be neater?

Reply:
I'm not sure. `PowerMethodNonsquare` is a `staticmethod`, i.e. a function which is a class member of `LinearOperator`. That means that you can call this class without an instance of a class (like `self` would be). 

Here Python is a bit confusing as **all its instance methods** are in facts static methods which receive the class instance. Have you ever wondered why the signature (the definition) of the methods in a class must have `self` as first parameter? That's the instance of the class. 

For this reason this works:

```python

class A(object):
    def methodA(self, parameter):
        print (parameter)

obj = A()
obj.methodA('hello world')
>>> hello world

A.methodA(obj, 'hello world')
>>> hello world
```
As a matter of facts, under the hood Python uses the second method when you type the first one. Therefore **in Python** there is actually no difference between my implementation and the one you are suggesting.